submissions/JOSEF/Alt-F4 - JOSEF.md

```diff
@@ -0,0 +1,156 @@
+
+Now there‘s only one little detail missing: We want JOSEF to actually make the materials that it‘s using! So all the remaining cells get dedicated to...
+
+## ...Production
+
+The production of building materials takes place in 2 different spaghetti cells. The story of how those get built is the similar to the ones before: When a station called “No Mining” is available and enough building materials are in the logistic network (and power isn't running low), a train comes in, deploys a blueprint and adds either “I” or “J” to the global constant combinator, depending on which production cell is built. This guy does one additional thing: It unloads a bit of coal (to make sure trains have something to run on) and, for production cell 2, some heavy oil barrels to start coal liquefaction.
+
+The builder train in charge simply alternates between the two different cells. There’s a simple “inserter cycle” that takes the blueprint book out of the train when it gets home, puts in the other one, then waits for the train to leave before activating again.
+
+![Production cell 1: Basic, non-oily things](https://i.imgur.com/BccZ1d9.png)
+
+The first cell produces all the different basic building materials: Belts, assemblers, miners, inserters, rails, trains, stations, signals, power poles – more or less everything that doesn‘t require oil. The basic production cell needs to be built before JOSEF runs out of stuff like belts or rails. This may be a problem if you have too many resource patches in the center or if you don’t give it a enough to start with and it decides to just solar panel the world. But you can always fix this by giving it a few more starting materials.
+
+  
+
+![Production cell 2: Advanced things using oil](https://i.imgur.com/CQsUf42.png)
+
+The second cell produces the more advanced things: Roboports, robots, logistic chests, blueprint deployers, solar panels & accumulators. This one needs to be built before JOSEF runs out of roboports. The builder train brings in 10 barrels of heavy oil which get emptied, run through the refinery and get refilled as soon as possible. JOSEF is not producing barrels but instead relying on the initial ones getting recycled infinitely.
+
+The two production cells are far from optimized, built to ratio or anything like that. They slowly and steadily produce all the things necessary but it may take an hour until a certain cell produces, say, their first train station, because it needs to fill up with belts first. However, it turns out that this becomes a non-issue after a bit, but more on that later.
+
+How many of these cells actually get supplied with ores depends on available resources, of course. Trains will always prefer to feed those cells close to resource patches so some further away cells might just stay empty for a while. But they will sooner or later get their resources, simply because there is an incredible number of...
+
+## ...Trains
```

Comment:
I think you should try incorporate this little section into the section of the production cells. It doesn't hold enough content to warrent it's own section :)